packages/frontend/editor-ui/src/features/dataStore/components/dataGrid/DataStoreTable.vue

```diff
@@ -529,6 +546,11 @@ const onCellValueChanged = async (params: CellValueChangedEvent<DataStoreRow>) =
 		await dataStoreStore.updateRow(props.dataStore.id, props.dataStore.projectId, id, {
 			[fieldName]: value,
 		});
+		telemetry.track('User edited data table content', {
+			data_table_id: props.dataStore.id,
+			column_name: fieldName,
```

Comment:
Shouldn't this be `column_id` instead?

Reply:
Do you mean we should substitute this with `id` or just change prop name (I doubt it's the second one but just checking)?

Reply:
The first. Changing `column_name: fieldName,` to `column_id: <the column-id>`

Reply:
Well, I'm thinking that our goal for this event is to track number of cell edits and as an extra number of cell edits per column for which we need to identify the column and the column_id sounds more correct value for this purpose. Tbh it's not a big deal so if there's no easy access to the column_id there or you think that column_name is more useful I'm fine with that.

Reply:
Fair point, `id` should be in the `colDef` so will add it here.